task_master.go

```diff
@@ -50,7 +50,7 @@ type LogService interface {
 type UDFService interface {
 	List() []string
 	Info(name string) (udf.Info, bool)
-	Create(name string, l *log.Logger, abortCallback func()) (udf.Interface, error)
+	Create(name, taskID, nodeID string, l *log.Logger, abortCallback func()) (udf.Interface, error)
```

Comment:
What's the motivation for exposing the task and node IDs? Is it just adding some additional nice to have information? Or something else?